.travis.yml

```diff
@@ -7,14 +7,14 @@ os:
   - osx
 
 rvm:
-  - "2.0.0"
-  - "2.1" # latest 2.1.x
-  - "2.2.5"
-  - "2.3.3"
-  - "2.4.0"
+  - "2.2.10"
```

Comment:
This is EOL - do we need to support this?  [[ref]](https://www.ruby-lang.org/en/downloads/branches/)